packages/server/src/transaction-context/transaction-context.ts

```diff
@@ -25,30 +21,41 @@ export interface ManageTransactionContextPropagator<T> extends TransactionContex
 
 export interface TransactionContextPropagator {
   /**
-   * EXPERIMENTAL: Transaction context propagation is experimental and subject to change.
-   * The OpenFeature Enhancement Proposal regarding transaction context can be found [here](https://github.com/open-feature/ofep/pull/32).
-   *
    * Returns the currently defined transaction context using the registered transaction
    * context propagator.
-   * @experimental
    * @returns {TransactionContext} The current transaction context
    */
   getTransactionContext(): TransactionContext;
 
   /**
-   * EXPERIMENTAL: Transaction context propagation is experimental and subject to change.
-   * The OpenFeature Enhancement Proposal regarding transaction context can be found [here](https://github.com/open-feature/ofep/pull/32).
-   *
    * Sets the transaction context using the registered transaction context propagator.
-   * @experimental
+   * Runs the {@link callback} function, in which the {@link transactionContext} will be available by calling
+   * {@link this#getTransactionContext}.
+   *
+   * The {@link TransactionContextPropagator} must persist the {@link transactionContext} and make it available
```

Comment:
Should we state context precedence in this doct?
I discovered from source code that flag evaluation passed context shadows keys set from other contexts

Reply:
Mh good question.
I think we could, on the other hand we are not doing it for all the other context variants right?
Do you want to propose something?

Reply:
We could include a link to https://openfeature.dev/specification/sections/evaluation-context#requirement-323

Reply:
Ya that's a good option @beeme1mr 🙂